### ISSUE TITLE
USAID healthcare facility location data

### ISSUE THREAD

**Issue by achafetz** [USAID-OHA-SI/find_your_beach#1]
C.Urbanowicz asking for 

> a list of all the cities where USAID has health care facilities and labs, so we can incorporate them into the map?

LPA looking to put together infographic maps for the NSC

**DUE: COB Thursday, April 9, 2020**

FYI @gsarfaty

**Comment by achafetz** [comment]
data pulled and now we have site information were available.

need to perform buffer around world cities

**Comment by tessam30** [comment]
Buffered finished by @baboyma. 
Sites intersected with city buffer and city info extracted. 
Take it home.

**Comment by achafetz** [comment]
Methodology - The request was around "where USAID has health care facilities and labs." We defined USAID health care facilities as any site a USAID implementing partner is working in that reports on testing or treatment and labs as any USAID supported laboratory-based testing and/or Point-of-Care Testing (POCT) sites engaged in continuous quality Improvement (CQI) and proficiency testing (PT) activities. These sites were mapped against a list of world cities. If any of these USAID sites fell within a 5km radius of the city center, we counted the city as a city USAID was working in.supporting. Our final city list can be filtered down depending on how large of a population you want to count as a large city; we recommend 40k.
